content/blog/2022-sefsc-summer.md

```diff
@@ -0,0 +1,75 @@
+
+<br> <center><a><img src="/img/blog/sefsc-sfd-github.png" width="900px" alt="alt txt"></a><figcaption>Example screenshot from the Sustainable Fisheries Division Assessment Workflow Team showing how they use GitHub for both an outward facing page and internal issue / progress tracking 😍</figcaption> </center> <br>
+
+### Creating and documenting a communication plan
+
+The Biology and Life History Branch within Fisheries, Assessment, Technology, and Engineering Support Division (FATES-BLH) documented their communication plan which acknowledged the areas where they are successful and identified areas for improvement (see figures below). Their branch has weekly meetings to strategize and track progress in real time, but is working to improve documentation of resulting products for easy and reproducible assessment inputs throughout SEFSC. Their team has been collaborating with SFD to receive assessment details through GitHub pages as well as collaborating within version-controlled programs, removing task redundancy and resulting in streamlined workflows. They are also working to document institutional knowledge and species-specific data issues, where the distribution of these details will vastly improve analyses and data inputs for stock assessments in the southeast. With SEFSC realignment, branches are spread throughout multiple cities in the southeast region, necessitating efficient communication and workflows.
+
+<br><center><a><img src="/img/blog/sefsc-fates-commsplan.png" width="900px" alt="alt txt"></a><figcaption>FATES-BLH Branch team communication plan which highlights current successes and areas for improvement.</figcaption></center><br>
+
+<br>
+
+<br> <center><a><img src="/img/blog/sefsc-fates-pathway.png" width="900px" alt="alt txt"></a><figcaption>FATES-BLH Openscapes team in their new pathway!  SEFSC Staff are spread throughout multiple cities in the southeast region, but Openscapes has facilitated streamlined communication for all staff.</figcaption></center><br>
```

Comment:
This png belongs below (next comment) - this one is the pathway illustration for FATES.